configs/components/openssl-3.0.rb

```diff
@@ -16,18 +16,18 @@
     pkg.build_requires 'perl'
   end
 
-  target = cflags = ldflags = sslflags = ''
+  target = sslflags = ''
+  cflags = settings[:cflags]
+  ldflags = settings[:ldflags]
 
- if platform.is_windows?
+  if platform.is_windows?
     pkg.environment 'PATH', "$(shell cygpath -u #{settings[:gcc_bindir]}):$(PATH)"
-  #   pkg.environment 'CYGWIN', settings[:cygwin]
-  #   pkg.environment 'CC', settings[:cc]
-  #   pkg.environment 'CXX', settings[:cxx]
-  #   pkg.environment 'MAKE', platform[:make]
+    pkg.environment 'CYGWIN', settings[:cygwin]
+    pkg.environment 'MAKE', platform[:make]
 
+#    cflags = "-I#{settings[:includedir]}"
```

Comment:
should these cflag and ldflag assignments be kept around, or do we plan on using them later?

Reply:
Ah good catch, I meant to remove that and forgot